BraintreeDropIn/BTDropInController.m

```diff
@@ -328,6 +323,7 @@ - (void)threeDSecureVerification:(BTPaymentMethodNonce *)tokenizedCard {
 }
 
 - (void)onLookupComplete:(__unused BTThreeDSecureRequest *)request result:(__unused BTThreeDSecureLookup *)result next:(void (^)(void))next {
+    [self.paymentSelectionViewController showLoadingScreen:NO];
```

Comment:
I believe this is executed before the completion block of `startPaymentFlow` is called. Do we need hide the loading screen in both places?

Reply:
Isn't this accounted for in line 309?

Reply:
There are a few different flows to consider:

1. Lookup completes successfully, but then the user hits cancel on the 3DS challenge screen, and they return to the payment selection screen. In that case, we don't want to show the spinner, so we hide it in `onLookupComplete`.

2. Lookup does not complete successfully, and `onLookupComplete` is not called. In that case, the `startPaymentFlow` completion block is called, so we need to hide the spinner at that point.

Reply:
I think that we call the `startPaymentFlow` completion block even if they cancel the 3DS flow, so just hiding it there should be sufficient.

Reply:
Hmmm... OK. I'll give that a try.

Reply:
Just pushed up that change 👍